docs/proposals/network-policy.md

```diff
@@ -176,14 +217,12 @@ def is_traffic_allowed(traffic, pod):
              ingress_rule.ports.matches(traffic):
           return True 
 
-  # network-isolation is True and no policies match the given pod and traffic.
+  # Ingress isolation is DefaultDeny and no policies match the given pod and traffic.
   return False
 ```
 
 ### Open Questions
 
-> The question then remains whether one NP object per "app" makes sense (in this case app={a,b,c}) or whether one NP object per selector
-
 - A single podSelector per NetworkPolicy may lead to managing a large number of NetworkPolicy objects, each of which is small and easy to understand on its own.  Allowing a podSelecor per ingress rule would allow for fewer, larger NetworkPolicy objects that may be easier to manage, but harder to understand individually.  This proposal has opted to favor a larger number of smaller objects that are easier to understand.  Is this the right decision? 
```

Comment:
Yep, I'll update.

I think there is fairly wide agreement to leave this as a single podSelector for now and see where that takes us, augmenting as you describe if the requirement arises.  The previous comment trail that indicated this has disappeared with the latest commit.